pages/blog/release-notes-2.3.0.md

```diff
@@ -0,0 +1,92 @@
+date: 2022-01-10T10:00:00+01:00
+type: Communication
+tags:
+  - Specification
+  - Release Notes
+cover: /img/posts/release-notes-2.2.0/cover.webp
+authors:
+  - name: Dale Lane
+    photo: /img/avatars/questionmark.webp
+    link: https://twitter.com/dalelane
+excerpt: 'AsyncAPI 2.3 is now released. This brings enhancements allowing new ways of structuring AsyncAPI documents and support for describing Solace APIs.'
+featured: true
+---
+
+The new version of the AsyncAPI specification - 2.3.0 - is now available.
+
+> This is a minor release, and it doesn't bring any breaking changes. You can switch to it by modifying the following value in your AsyncAPI file `asyncapi: '2.2.0'` into `asyncapi: '2.3.0'`
+
+
+## Servers and channels can now be defined as reusable components
+
+To allow for more flexibility in how AsyncAPI documents are structured and enable content to be reused, `servers` and `channels` can now be defined as reusable components.
+
+For example:
+
+```yaml
+asyncapi: 2.3.0
+servers:
+  production:
+    $ref: '#/components/servers/myserver'
+channels:
+  some/events:
+    $ref: '#/components/channels/myChannel'
+components:
+  servers:
+    myserver:
+      url: "http://localhost:5000/ws"
+      protocol: ws
+  channels:
+    myChannel:
+      description: "mychannel"
+```
+
+These are added to the many other aspects of the AsyncAPI specification which can be declared as reusable components. You can see the full list in the [Components Object section of the AsyncAPI specification](https://www.asyncapi.com/docs/specifications/v2.3.0#componentsObject).
+
+This new feature was contributed by [Sergio Moya](https://www.linkedin.com/in/smoya). For more detail, see this [pull request](https://github.com/asyncapi/spec/pull/665) and the [Github issue where this change was discussed](https://github.com/asyncapi/spec/issues/660).
+
+
+## New protocol bindings
+
+The specification is now extended to support another custom protocol through the bindings feature:
+
+Solace, thanks to [Michael Davis](https://github.com/damaru-inc).
+For more details, check out this [pull request](https://github.com/asyncapi/spec/pull/666) and [binding definition](https://github.com/asyncapi/bindings/tree/master/solace).
+
+
+## Other enhancements
+
+Regular expressions are now Unicode-compliant. This enhancement to the specification from [Sergio Moya](https://www.linkedin.com/in/smoya) means that regular expressions (found in `pattern` and `patternProperties` fields) are now Unicode-compliant (according to ECMA-262).
+
+This improves compatibility with some JSON Schema parsers. For more details, check out the [pull request](https://github.com/asyncapi/spec-json-schemas/pull/145).
+
+
+## Deprecations
+
+The `$ref` field in [Channel Item Object](#https://www.asyncapi.com/docs/specifications/v2.3.0#channelItemObject) is deprecated from AsyncAPI 2.3.0, in favour of using [Reference Object](https://www.asyncapi.com/docs/specifications/v2.3.0#referenceObject) instead.
```

Comment:
Hi @dalelane,

Would it be possible to rephrase this section? I've read it couple of times and the take away (in my understanding) is that the `Channel Item`.`$ref` field is deprecated and `Reference Object` should be used instead.

That's actually not the case. IMHO we should emphasize that though the `Channel Item`.`$ref` is now deprecated, it's still the only mechanism for referencing Channel Item Objects. `Reference Object` cannot be used in 2.3 version of spec to reference `Channel Item Object` as it would introduce ambiguity described in https://github.com/asyncapi/spec/issues/607#issuecomment-1018318910. Basically we're deprecating something that doesn't have other preferred way of substitution until 3.0 version.

In 3.0, the `Channel Object`.`$ref` will be dropped and the `Reference Object` will be allowed to be used wherever the `Channel Item Object` can. But this is not the case for version 2.3.

Reply:
absolutely - thanks so much for checking it over and clarifying 👍